plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/editors/data/preferences/PrefPageResultSetMain.java

```diff
@@ -293,7 +319,9 @@ protected void loadPreferences(DBPPreferenceStore store)
             }
             showErrorsInDialog.setSelection(store.getBoolean(ResultSetPreferences.RESULT_SET_SHOW_ERRORS_IN_DIALOG));
             markCellValueOccurrences.setSelection(store.getBoolean(ResultSetPreferences.RESULT_SET_MARK_CELL_VALUE_OCCURRENCES));
-
+            int maxEditiingContentSize = store.getInt(ModelPreferences.EDITING_CONTENT_MAX_SIZE_KBYTES);
+            maxEditingContentSize.setText(String.valueOf(maxEditiingContentSize));
```

Comment:
Redundant local variable

```suggestion
maxEditingContentSize.setText(String.valueOf(store.getInt(ModelPreferences.EDITING_CONTENT_MAX_SIZE_KBYTES)));
```

Reply:
The variable extracted for clarity.